src/js/media-captions-button.js

```diff
@@ -112,6 +114,9 @@ class MediaCaptionsButton extends MediaChromeButton {
   }
 
   handleClick() {
+    // do nothing if parent is an instance of MediaChromeSelectMenu
+    if (this.parentElement instanceof MediaChromeSelectMenu) return;
```

Comment:
It's needed because otherwise the button will do its default behavior, which we don't want in this case. Happy to do the check in some other way, such as getting the select-menu from the custom elements registry or something.

Reply:
yes, we need to stop propagation of the click event but from outside this button, not inside

Reply:
Unfortunately, it won't work. The click listener here is registered earlier than the one in media-chrome-select-menu, so, even if we call `preventDefault` or `stopPropagation` or even `stopImmediatePropagation`, this method had already run.

Reply:
I updated this to use `window.customElements.get('media-chrome-select-menu')` rather than depend on it directly so that the other files won't get included if someone is importing just media-captions-button.